src/components/Popups/PopupReply.tsx

```diff
@@ -64,11 +69,20 @@ export default function PopupReply({
     ...rest
 }: Props): JSX.Element {
     const popupRef = React.useRef<PopupBase>(null);
+    const popupOptions = React.useRef<Partial<Popper.Options> | null>(null);
     const rotation = ReactRedux.useSelector(getRotation);
     const prevRotation = usePrevious(rotation);
     const scale = ReactRedux.useSelector(getScale);
     const prevScale = usePrevious(scale);
 
+    // Keep the options reference the same between renders
+    if (!popupOptions.current) {
```

Comment:
Should this be in a React.useEffect call that only occurs on mount?

Reply:
Good idea I'll try that

Reply:
Don't think that works because the hook runs after render and as a result the placement gets misplaced a bit